src/main/resources/com/mesosphere/velocity/marathon/MarathonStep/config.jelly

```diff
@@ -6,10 +6,16 @@
     <f:entry field="forceUpdate" title="${%Force Update}">
         <f:checkbox name="forceUpdate" />
     </f:entry>
-    <f:entry field="appid" title="${%App ID}">
+    <f:entry field="appid" title="${%Application Id}">
         <f:textbox/>
     </f:entry>
     <f:entry field="docker" title="${%Docker}">
         <f:textbox/>
     </f:entry>
+    <f:entry field="credentialsId" title="${%Authentication Credentials}">
+        <f:textbox/>
```

Comment:
This should be `<c:select>`, similar to the credentials in the [git plugin](https://github.com/jenkinsci/git-plugin/blob/master/src/main/resources/jenkins/plugins/git/GitStep/config.jelly). This will also need a matching `doFillCredentialsIdItem` method on the step descriptor ([git plugin example](https://github.com/jenkinsci/git-plugin/blob/master/src/main/java/jenkins/plugins/git/GitStep.java#L95-L99)).

Does the `textbox` style work now? I can make the switch to `select` in a later pull request.